test/io.test.js

```diff
@@ -170,18 +188,24 @@ describe('IO', function(){
         describe('readLong()', function(){
             it('should decode and return a long', function(){
                 block.write(new Buffer([0x94, 0x02]));
-                decoder.readLong().should.equal(138);
+                decoder.readLong().equals(138).should.be.true;
```

Comment:
This seems like a topic of discussion for the Long.js repo (see https://github.com/dcodeIO/Long.js).  If valueOf is implemented for that library, we can revisit this discussion.  However, without that implementation, I believe what I've written is how it needs to be done for now.